functions/src/utils/firebase.js

```diff
@@ -13,6 +13,7 @@ const config = {
 admin.initializeApp({
     credential: admin.credential.cert(serviceAccount),
     databaseURL: databaseURL,
+    storageBucket: storageBucket,
```

Comment:
given that the key and value are the same you can leave off the value
```
...
databaseURL,
storageBucket,
...
```

Reply:
Okay. Wil do that